test/benchmark/iam/LanguageFeatureTest.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.core.reasoner.benchmark.iam;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class LanguageFeatureTest{
+
+    private static final String database = "iam-benchmark-language-features";
+    private static final BenchmarkRunner benchmarker = new BenchmarkRunner(database);
+
+    public LanguageFeatureTest() { }
+
+    @BeforeClass
+    public static void setup() throws IOException {
+        benchmarker.setUp();
+        benchmarker.loadSchema("schema_types.tql");
+        benchmarker.loadSchema("schema_rules_optimised.tql");
+        benchmarker.loadSchema("schema_rules_test_specific.tql");
```

Comment:
is there way to factor these schema files into a set of "common" types and rules, and then have one set of rules for `schema_rules_language_feature.tql` which matches this test name, then another for `conjunction_structure`, etc.?